pkgs/tools/misc/pre-commit/default.nix

```diff
@@ -82,7 +78,8 @@ buildPythonPackage rec {
   preCheck = ''
     export GIT_AUTHOR_NAME=test GIT_COMMITTER_NAME=test \
            GIT_AUTHOR_EMAIL=test@example.com GIT_COMMITTER_EMAIL=test@example.com \
-           VIRTUALENV_NO_DOWNLOAD=1 PRE_COMMIT_NO_CONCURRENCY=1 LANG=en_US.UTF-8
+           VIRTUALENV_NO_DOWNLOAD=1 PRE_COMMIT_NO_CONCURRENCY=1 LANG=en_US.UTF-8 \
+           RUSTFLAGS="-Ctarget-feature=-outline-atomics"
```

Comment:
It’s not clear to me why this was added. I would either add a comment or add an explanation to the body of the commit message.